content/en/coec/index.adoc

```diff
@@ -0,0 +1,53 @@
+---
+title: "Code of Ethics and Conduct"
+description: "ACL Special Interest Group on Turkic Languages Code of Ethics and Conduct"
+draft: false
+toc: true
+images: []
+contributors: []
+---
+
+:toc:
+
+== Introduction
+
+SIGTURK is an ACL Special Interest Group on Turkic Languages that aims to foster community brainstorming and collaboration on events and resources for the public good in open source. As these processes involve sync and async communication, this document includes professional standards all participants of SIGTURK and SIGTURK-related dialogue must uphold to promote formal reliability of SIGTURK outputs, respect, competence, responsibility, and integrity while recognizing the rapidly changing society and the necessity to exercise more considerable caution in this spirit by every participant.
```

Comment:
Some suggestions (feel free to discuss):

- Isn't SIGTURK _the_ ACL Special Interest Group on Turkic Languages? :)  
- "in open source" -> "in an open source manner" (or "fashion") ? 
- "this document includes" -> "this document specifies"
- "must uphold to promote..." -> "agree to uphold in order to promote..."